index.html

```diff
@@ -43,9 +43,146 @@ <h1>Hey there. I’m Salim.<br><span class="highlight">I’m a software develope
+                    </div>
+                </li>
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
+                    <ul class="tags">
+                        <li class="tag">Ruby on rails</li>
+                        <li class="tag">css</li>
+                        <li class="tag">JavScript</li>
+                        <li class="tag">html</li>
+                    </ul>
+                    <div class="button-container">
+                        <button type="button">See More</button>
+                    </div>
+                </li>
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
+                    <ul class="tags">
+                        <li class="tag">Ruby on rails</li>
+                        <li class="tag">css</li>
+                        <li class="tag">JavScript</li>
+                        <li class="tag">html</li>
+                    </ul>
+                    <div class="button-container">
+                        <button type="button">See More</button>
+                    </div>
+                </li>
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
+                    <ul class="tags">
+                        <li class="tag">Ruby on rails</li>
+                        <li class="tag">css</li>
+                        <li class="tag">JavScript</li>
+                        <li class="tag">html</li>
+                    </ul>
+                    <div class="button-container">
+                        <button type="button">See More</button>
+                    </div>
+                </li>
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
+                    <ul class="tags">
+                        <li class="tag">Ruby on rails</li>
+                        <li class="tag">css</li>
+                        <li class="tag">JavScript</li>
+                        <li class="tag">html</li>
+                    </ul>
+                    <div class="button-container">
+                        <button type="button">See More</button>
+                    </div>
+                </li>
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
```

Comment:
- [ ] great job on the structure of your code 👍 according to the requirements however this title should be a header tag `<hx>` instead of a paragraph `<p>`